src/pages/error/error.jsx

```diff
@@ -0,0 +1,32 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash" className="error">
+        <div id="masthead">
+          <div className="inner">
+            <div id="header">
+              <a href="#/"><img src="./img/logo.png" width="200" height="53" alt="Webmaker" /></a>
```

Comment:
I assume they're coming from the design? It's a logo so I'm not sure why hardcoding this value is an issue. It's either hardcoded here or in the CSS.